include/boost/geometry/strategies/geographic/distance_cross_track.hpp

```diff
@@ -62,9 +62,37 @@ namespace boost { namespace geometry
 
 namespace strategy { namespace distance
 {
-
 namespace detail
 {
+
+template <bool>
```

Comment:
I propose to add a tparam name here to indicate what's happening. This would be `EnableClosestPoints` right?

Btw, if this utility alwasy has to take distance and coordinates of the closest point then it means that all of these always exist. If they exist they has to be calculated. So what is the purpose of assigning them conditionally if they are always there anyway? Or is this only the case in geographic CS?

Reply:
Exactly, in geographic (unlike in spherical) computing distance and closest points is the same computation so `EnableClosestPoints` is only to avoid some (few) extra assignments. I do not mind removing it if you prefer.